manifests/init.pp

```diff
@@ -67,6 +64,7 @@
     ensure  => 'file',
     mode    => '0644',
     content => $_real_conf,
+    notify  => Service['NetworkManager'],
```

Comment:
Please add spec coverage of this relationship.

Reply:
I have extended the spec coverage.